test/shared/src/main/scala/zio/test/TestAspect.scala

```diff
@@ -280,6 +281,25 @@ object TestAspect extends TimeoutVariants {
    */
   val sequential: TestAspectPoly = executionStrategy(ExecutionStrategy.Sequential)
 
+  /**
+   * An aspect that reports the execution time of tests.
+   */
+  val timed: TestAspect[Nothing, Live[Clock] with Live[Console], Nothing, Any, Nothing, Any] =
+    new TestAspect.PerTestLabeled[Nothing, Live[Clock] with Live[Console], Nothing, Any, Nothing, Any] {
+      def perTestLabeled[R <: Live[Clock] with Live[Console], E, S, L](
+        label: L,
+        test: ZIO[R, TestFailure[E], TestSuccess[S]]
+      ): ZIO[R, TestFailure[E], TestSuccess[S]] =
+        withLive(test.either)(_.timed).flatMap {
+          case (duration, ees) =>
+            live {
+              console.putStrLn {
+                s"$label took ${duration.toMillis / 1000.0} seconds"
```

Comment:
I noticed we have similar code to render durations in `TimeoutVariants#renderDuration` and `DefaultTestReporter#logStats`. How about extracting it and using the same one everywhere? `TimeoutVariants` is probably the best one.

Reply:
Good idea. Will do that tomorrow.